packages/navigator/src/browser/navigator-widget.tsx

```diff
@@ -159,19 +159,23 @@ export class FileNavigatorWidget extends FileTreeWidget {
     protected handleCopy(event: ClipboardEvent): void {
         const uris = this.model.selectedFileStatNodes.map(node => node.uri.toString());
         if (uris.length > 0) {
```

Comment:
Could we reuse the same `if` statement ?
That way it'd be:

```ts
if (uris.length > 0 && event.clipboardData) { ... }
```

Reply:
Yes, I prefer that way too, but I didn't use it because I wanted to keep the same style used in `problem-widget`
https://github.com/theia-ide/theia/blob/7ee2997ac226d53733bb25749dc9d25d43bac5aa/packages/markers/src/browser/problem/problem-widget.tsx#L66-L71

What should I do? Change that `if` it in both places or just here?

Reply:
Ahh I see, I wouldn't mind if you update both places :)